src/DotNetWorker.Core/Logging/ILoggerExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.Azure.Functions.Worker.Logging.ApplicationInsights;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Extensions for <see cref="ILogger"/>.
+    /// </summary>
+    public static class ILoggerExtensions
+    {
+        /// <summary>
+        /// Logs a metric value.

```

Comment:
Should we explicitly specify that we will be logging as Information level message? Either in the method comment/documentation or the method name?